mne/time_frequency/tfr.py

```diff
@@ -1065,7 +1065,7 @@ def save(self, fname, overwrite=False):
 def _prepare_write_tfr(tfr, condition):
     """Aux function"""
     return (condition, dict(times=tfr.times, freqs=tfr.freqs,
-                            data=tfr.data, info=tfr.info, nave=tfr.nave,
+                            data=tfr.data, info=tfr.info._store, nave=tfr.nave,
```

Comment:
Down the line `write_hdf5` will complain that it doesn't know how to write an object of that type, because `isinstance(Info, dict) is False` now. You have to pass a dict-like object for the purposes of writing using `write_hdf5`.